### PR TITLE
Updating imu.ino

### PR DESCRIPTION
**Pull request by Akettle44**
Updating SSA development with IMU example. There is a separate IMU folder for independently testing the IMU.

### CODE REVIEW

File: Embedded_Code/SSA/imu/imu.ino [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+//Reading IMU
+  getI2CData();
+  //delay(1000);
+  Serial.flush();
+}
+
+void I2C_init()
+{
+  pinMode(scl, INPUT_PULLUP);
+  pinMode(sda, INPUT_PULLUP);
```
Comment (LucasJuttner) [inline Embedded_Code/SSA/imu/imu.ino:65]: move to setup function

File: Embedded_Code/SSA/imu/imu.ino [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+  Wire.write(accelX0); //getting data from accelerometer registers
+  Wire.write(accelX1);
+  Wire.write(accelY0);
+  Wire.write(accelY1);
+  Wire.write(accelZ0);
+  Wire.write(accelZ1);
+
+  Wire.endTransmission(true); //continue transmission until reading is done
+
+  Wire.requestFrom(lsm9ds1_ag, 24); //requesting 12 bytes, or 3 16 bit numbers.
```
Comment (LucasJuttner) [inline Embedded_Code/SSA/imu/imu.ino]: for comment on this line: 12 bytes != 3,16bit numbers so i am confusion

**Comment by LucasJuttner** [comment]
Can you get rid of everything else(ir sensors and hall effect) that doesnt have to do with the IMU for this file?

File: Embedded_Code/SSA/imu/imu.ino (hunk 2 above)
Comment (Akettle44) [reply Embedded_Code/SSA/imu/imu.ino]: Old comment, will update

File: Embedded_Code/SSA/imu/imu.ino (hunk 1 above)
Comment (Akettle44) [reply Embedded_Code/SSA/imu/imu.ino:65]: I2C_init() is being called in setup, do you still want me to move it?

**Pull request merged**